store/src/java/com/zimbra/cs/imap/ImapDaemon.java

```diff
@@ -98,15 +99,20 @@ public static void main(String[] args) {
             try (FileInputStream fisLog4j = new FileInputStream(IMAPD_LOG4J_CONFIG)) {
                 props.load(fisLog4j);
             }
-            PropertyConfigurator.configure(props);
+            try {
+                ConfigurationSource logConfigSource = new ConfigurationSource(new FileInputStream(LC.zimbra_log4j_properties.value()));
```

Comment:
It looks like this now uses a different FileInputStream than it used to (see line 99 just above).

Reply:
Do we still need to create the `props` variable above?